### DISCUSSION TITLE
Configure telescope without making changes in the core package

### DISCUSSION THREAD

**Discussion by vshalt** [LazyVim/LazyVim#2191] [General]
How do I configure telescope hide/ignore files config without making changes in the core package

**Comment by dpetka2001** [comment]
What do you want to do exactly?

**Reply by vshalt** [reply]
I want to hide/ignore files and folders. If I am working a git repository I usually put them in the gitignore file.

But outside a git repository I get suggestions from node_modules

**Reply by dpetka2001** [reply]
Telescope has an option `file_ignore_patterns`, which you can use to put patterns that you want to ignore when using Telescope pickers (if you decide to use this option, be aware that this also affects the LSP pickers, so you won't have any LSP information from the folders that you will include here). 
Alternatively, if you have `ripgrep` installed, you can also use a `.ignore` file with patterns inside to ignore, since `rg` is usually the first option for Telescope pickers if it's installed.

**Reply by vshalt** [reply]
Where do I put this config in? Should I be modifying the lazyvim folder in ~/.local/share/nvim/lazyvim?

**Reply by dpetka2001** [reply]
No you shouldn't. Plz read the [docs](https://www.lazyvim.org/configuration/plugins#-adding-plugins). They contain most basic information about things like these.
